mne/time_frequency/tfr.py

```diff
@@ -1265,20 +1253,20 @@ def plot_joint(self, picks=None, baseline=None, mode='mean',
         exclude : None | list of str | 'bads'
             Channels names to exclude from being shown. If 'bads', the
             bad channels are excluded. Defaults to None.
-        timefreqs : None | iterable of tuples
+        timefreqs : None | list of tuples | dict of tuples
             The time-frequency point(s) to plot. Each tuple defines a pair
-            of a time (in s) and a frequency (in Hz).
-        timefreqs_average : None | tuple | iterable of tuples
-            The time and/or frequency window around a given timefreq point
-            to be used for averaging (seconds, Hertz). If a tuple is
-            provided, use the same window span for all timefreqs points
-            defined in `timefreqs`.For example, (0.01, 2) would translate
-            into a window that starts 5 ms before and ends 5 ms after a
-            given time point, as well as spanning 1 Hz before and after the
-            given frequency point.
-            If an iterable of tuples of the same length as `timefreqs` is
-            passed, then each tuple applies to its corresponding timefreqs
-            point. Defaults to None, which means no averaging.
+            of a time (in s) and a frequency (in Hz). If provided as a
+            dictionary, these tuples are keys, while their corresponding
+            values are also tuples, indicating the time and/or frequency
+            window around a given timefreq point to be used for averaging.
+            For example,
+
+                timefreqs={(1, 10): (0.1, 2)}
```

Comment:
From [here](https://docs.python.org/2/tutorial/datastructures.html#dictionaries), it looks like this only works if the keys are strings. We could do it like this though (but it's more cluttered):
```
timefreqs=dict([((1, 10), (0.1, 2))])
```